### DISCUSSION TITLE
Willing to help..

### DISCUSSION THREAD

**Discussion by jasonk** [faker-js/faker#55] [General]
It looks like you are attempting to turn this into a community-controlled package, which is a great idea given recent events.  Since I've been using it at my day job I'd be willing to pitch in and help.  FWIW I'm already mentioned in the README, as I wrote the Perl Data::Faker package that it's based on...

**Comment by damienwebdev** [comment]
@jasonk thank you for the offer. Your copyright has been in this repo for a very long time, so having your background is most appreciated. If you use Discord, we have a nice little community server going - #21 Otherwise, I'd love to get any insight from you in general!

**Comment by quibbler01** [comment]
Just woke up this morning and saw the news. It seems because the original author wanted to commercialize the library and failed.

**Comment by damienwebdev** [comment]
Hi @jasonk , I never thought to ask, so apologies for that, but we're considering sticking with the name "Faker" for the future of this project. I believe you're the original copyright holder to that name, are you ok with us utilizing the name in this project?

**Reply by jasonk** [reply]
I'm fine with that, though the original was actually called `Data::Faker` as the Perl community frowned on top-level namespaces at the time..
